src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2696,6 +2696,19 @@ public static void IdsConversionsTest(string windowsId, string ianaId)
         [InlineData("Central Europe Standard Time", "Europe/Tirane", "AL")]
         [InlineData("Central Europe Standard Time", "Europe/Podgorica", "ME")]
         [InlineData("Central Europe Standard Time", "Europe/Belgrade", "RS")]
+        // lowecased region name cases:
```

Comment:
Nit: typo in comment

```suggestion
        // lowercased region name cases:
```